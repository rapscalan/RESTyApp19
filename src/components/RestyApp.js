import React, { Component } from 'react';


export default class RestyApp extends Component {
  // state = {

  // } 
  render() {
    return <h1>Hello World</h1>;
  }
}
