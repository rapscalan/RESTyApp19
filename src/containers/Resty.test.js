import React from 'react';
import { shallow } from 'enzyme';
import RestyApp from './Resty';

describe('RestyApp component', () => {
  it('renders RestyApp', () => {
    const wrapper = shallow(<RestyApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
