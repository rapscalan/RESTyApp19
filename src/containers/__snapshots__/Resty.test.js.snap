// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RestyApp component renders RestyApp 1`] = `
<Fragment>
  <Requests
    requests={Array []}
  />
</Fragment>
`;
